src/vs/workbench/contrib/positronWelcome/browser/helpers.ts

```diff
@@ -0,0 +1,98 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2025 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { URI } from '../../../../base/common/uri.js';
+import { ICommandService } from '../../../../platform/commands/common/commands.js';
+import { INotificationService, NeverShowAgainScope, Severity } from '../../../../platform/notification/common/notification.js';
+import { IStorageService, StorageScope, StorageTarget } from '../../../../platform/storage/common/storage.js';
+import { IPathService } from '../../../services/path/common/pathService.js';
+import { PositronImportSettings } from './actions.js';
+import * as platform from '../../../../base/common/platform.js';
+import { localize } from '../../../../nls.js';
+
+const wasPromptedKey = 'positron.welcome.promptedImport';
```

Comment:
```suggestion
const WAS_PROMPTED_KEY = 'positron.welcome.promptedImport';
```

We typically use all caps for constants like this. Can you update this value?